### PR TITLE
Support for cross-repository PRs in same github organization

### PR DESCRIPTION
**Pull request by RadekManak**
When GitHub fork are created in the same organization, the full repository name must be specified in newly added `head_repo` parameter to the create pull request API call. 

This API field is not yet implemented in the library we use. I've split this PR into two commits. First, that uses a not yet merged library function and second commit that uses of `_post` internal library function to make the create API call for now.

### CODE REVIEW

**Comment by RadekManak** [comment]
Verified here:
https://github.com/openshift-cloud-team/topolvm/pull/6
Does not break normal forks here:
https://github.com/openshift-cloud-team/topolvm/pull/7

**Review by damdo — COMMENTED** [review]
/approve

Thanks @RadekManak 

cc. @elmiko for LGTM

File: rebasebot/bot.py [hunk 1]
```diff
@@ -439,14 +439,17 @@ def _is_push_required(gitwd: git.Repo, dest: GitHubBranch, source: GitHubBranch,
...
+def _is_pr_available(dest_repo: Repository, dest: GitHubBranch, rebase: GitHubBranch) -> Tuple[ShortPullRequest, bool]:
     logging.info("Checking for existing pull request")
-    try:
-        gh_pr = dest_repo.pull_requests(head=f"{rebase.ns}:{rebase.branch}").next()
-        logging.info("Found existing pull request: %s", gh_pr.html_url)
-        return gh_pr, True
-    except StopIteration:
-        pass
+
+    pull_requests = dest_repo.pull_requests(base=dest.branch)
```
Comment (elmiko) [inline rebasebot/bot.py:445]: do we no longer need to catch the `StopIteration` exception?

**Review by elmiko — COMMENTED** [review]
looks good to me, just a quick question

File: rebasebot/bot.py (hunk 1 above)
Comment (RadekManak) [reply rebasebot/bot.py:445]: No, instead of calling .next() that trows StopIteration, we for loop over the slice.

**Pull request merged**